docs/api/components/schemas.yaml

```diff
@@ -1634,6 +1634,13 @@ Database:
       description: 'The amount of milliseconds a N1QL query should run before logging a warning. '
       type: number
       default: 500
+    store_legacy_revtree_data:
+      description: |-
+        Controls whether Sync Gateway stores additional legacy revision tree pointer data to support 3.x/early 4.x clients that still use RevTree IDs (for example when used as delta sources).
```

Comment:
Is there any case other than delta sync where this is relevant?

I might phrase this in closer terms to functional impact. I'm not fussed on the particularly wording, take or leave this.

```
Controls whether Sync Gateway stores a mapping per document to map between a Sync Gateway < 4.0 revision ID and a Sync Gateway 3.0 revision ID.

This can be turned to false if delta_sync is disabled, or all clients (Couchbase Lite, Inter-Sync Gateway Replication, Sync Gateway) have documents that have been modified by Sync Gateway or Couchbase Lite 4.0+
```

If this only affects delta sync, should the parameter be tied to delta sync? Probably not, since you might lose the ability to enable delta sync after a document has been replicated.